OmniKitTests/PodInfoTests.swift

```diff
@@ -225,15 +225,15 @@ class PodInfoTests: XCTestCase {
             XCTAssertEqual(.faultEvents, decoded.podInfoType)
             XCTAssertEqual(.errorEventLoggedShuttingDown, decoded.podProgressStatus)
             XCTAssertEqual(.suspended, decoded.deliveryStatus)
-            XCTAssertEqual(0000, decoded.insulinNotDelivered)
+            XCTAssertEqual(0, decoded.insulinNotDelivered)
             XCTAssertEqual(4, decoded.podMessageCounter)
-            XCTAssertEqual("07eb", decoded.unknownPageCode.hexadecimalString)
+            XCTAssertEqual("50.15", decoded.totalInsulinDelivered.twoDecimals)
```

Comment:
Probably better to test actual value here, than string representation.  The .twoDecimals method is a private extension for formatting the debugDescription, not for external use.